### ISSUE TITLE
Use only 4 pokestop

### ISSUE THREAD

**Issue by Trenipuccio** [finndev/PokeBuddy#474]
When i use Pokebuddy, my character stay only in 4 pokestop for all the time that i use the bot. It's a bug?

**Comment by pokemonloverx333** [comment]
maybe u got a bad spot to farm , where are no other pokestops nearby

**Comment by Trenipuccio** [comment]
Central Park is my spot

**Comment by metno7** [comment]
This might be the same issue as #173 that I experience as well. Do you use a PTC or Google account?

**Comment by Trenipuccio** [comment]
I use PTC. My friend who uses my PTC account to see if the same mistake repeats, but he doesn't have this problem. @metno7

**Comment by metno7** [comment]
Yes you're right, it's a client side problem. I could fix it on my side by running Pokebuddy with admin privileges (right click -> run as administor). does that also work for you?

**Comment by Trenipuccio** [comment]
I'll try, in this pc i cannot use admin privileges, but when i'm at home i'll try. Ty for the help
